test/e2e/storage/persistent_volumes-local.go

```diff
@@ -237,9 +485,26 @@ func createLocalPVCPV(config *localTestConfig, volume *localTestVolume) {
 	var err error
 	volume.pv, volume.pvc, err = framework.CreatePVPVC(config.client, pvConfig, pvcConfig, config.ns, true)
 	framework.ExpectNoError(err)
-	framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc)
+	framework.ExpectNoError(framework.WaitOnPVandPVC(config.client, config.ns, volume.pv, volume.pvc))
 }
 
-func createLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
+func makeLocalPod(config *localTestConfig, volume *localTestVolume, cmd string) *v1.Pod {
 	return framework.MakePod(config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, cmd)
 }
+
+func createLocalPod(config *localTestConfig, volume *localTestVolume) (*v1.Pod, error) {
+	return framework.CreatePod(config.client, config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, "")
+}
+
+func createWriteAndReadCmds(testFileDir string, testFile string, writeTestFileContent string) (string, string) {
+	testFilePath := fmt.Sprintf("%s/%s", testFileDir, testFile)
+	writeCmd := fmt.Sprintf("mkdir -p %s; echo %s > %s; cat %s", testFileDir, writeTestFileContent, testFilePath, testFilePath)
```

Comment:
`writeCmd` also reads the file. Maybe `writeReadCmd` is a better name. Or maybe it should just write to the file?